src/common/Card.ts

```diff
@@ -1,7 +1,78 @@
-import { getAnimations } from "../getStyles.js";
-import { encodeHTML, flexLayout } from "./utils.js";
+import { getAnimations } from "../getStyles";
+import { encodeHTML, flexLayout } from "./utils";
 
-class Card {
+/** Card colors. */
+export interface CardColors {
+  /** Title color. */
```

Comment:
All these comments are quite repetitive, the property itself is very self explanetory.